src/Adapter/XmlseclibsAdapter.php

```diff
@@ -65,6 +65,16 @@ class XmlseclibsAdapter implements AdapterInterface
      */
     protected $transforms = [];
 
+    /**
+     * Create the XLMSecurityDSign class
+     *
+     * @return new XMLSecurityDSig()
+     */
+    protected function createXmlSecurityDSig($options = array()) {
```

Comment:
The argument $options is not used. Please remove.

Also move the protected method after public methods.